src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -169,12 +168,15 @@ def _initialize_scale_zero_point(
     if quantization_args.dynamic:
         return
 
-    device = next(module.parameters()).device
-    if is_module_offloaded(module):
-        device = get_execution_device(module)
+    # begin on the same device as other parameters or cpu if offloaded
+    params_device = next(module.parameters()).device
+    device = "cpu" if has_offloaded_params(module) else params_device
 
     # infer expected scale/zero point shape
-    expected_shape = 1  # per tensor
+    if quantization_args.strategy == QuantizationStrategy.TOKEN:
+        expected_shape = (1, 1)
```

Comment:
Just for my understanding, why is the shape (1,1) here when used for Param init for token level quant?

Reply:
See #210. This is the correct shape for per-token strategy